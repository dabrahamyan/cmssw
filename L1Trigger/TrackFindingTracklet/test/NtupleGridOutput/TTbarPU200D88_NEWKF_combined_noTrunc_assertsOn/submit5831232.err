************************************************************************************************************************
						 WARNING!
 The key phase2_realistic now points to 125X_mcRun4_realistic_v5.
 Usage of this key is discretionary and users are cautioned to use it at their own risk!
 This Global Tag contains tags suited for an Inner Tracker layout with rectangular (25x100um2) pixel cells (T15,T21).
 If this is not the geometry you are expecting to use, please consult:
 https://github.com/cms-sw/cmssw/blob/master/Configuration/AlCa/python/autoCondPhase2.py to retrieve the right key.
************************************************************************************************************************
04-Sep-2023 10:43:42 CEST  Initiating request to open file root://cms-xrd-global.cern.ch///store/mc/CMSSW_12_6_0/RelValTTbar_14TeV/GEN-SIM-DIGI-RAW/PU_125X_mcRun4_realistic_v5_2026D88PU200RV183v2-v1/30000/4a62553b-ce6d-451a-865b-b50023fc4966.root
04-Sep-2023 10:43:53 CEST  Successfully opened file root://cms-xrd-global.cern.ch///store/mc/CMSSW_12_6_0/RelValTTbar_14TeV/GEN-SIM-DIGI-RAW/PU_125X_mcRun4_realistic_v5_2026D88PU200RV183v2-v1/30000/4a62553b-ce6d-451a-865b-b50023fc4966.root
%MSG-w XrdAdaptorLvl3:  AfterFile 04-Sep-2023 10:44:08 CEST pre-events
Deactivating heplnx229.pp.rl.ac.uk from active sources because its quality (776) is higher than 260 and 4 times larger than the other active server heplnx228.pp.rl.ac.uk (18) 
%MSG
L1TrackNtupleMaker::beginJob
Begin processing the 1st record. Run 1, Event 4101, LumiSection 42 on stream 0 at 04-Sep-2023 10:44:27.684 CEST
Begin processing the 2nd record. Run 1, Event 4102, LumiSection 42 on stream 0 at 04-Sep-2023 10:44:33.743 CEST
Begin processing the 3rd record. Run 1, Event 4104, LumiSection 42 on stream 0 at 04-Sep-2023 10:44:38.483 CEST
Begin processing the 4th record. Run 1, Event 4103, LumiSection 42 on stream 0 at 04-Sep-2023 10:44:44.957 CEST
Begin processing the 5th record. Run 1, Event 4106, LumiSection 42 on stream 0 at 04-Sep-2023 10:44:51.158 CEST
Begin processing the 6th record. Run 1, Event 4105, LumiSection 42 on stream 0 at 04-Sep-2023 10:44:56.124 CEST
Begin processing the 7th record. Run 1, Event 4107, LumiSection 42 on stream 0 at 04-Sep-2023 10:45:01.303 CEST
Begin processing the 8th record. Run 1, Event 4108, LumiSection 42 on stream 0 at 04-Sep-2023 10:45:07.615 CEST
Begin processing the 9th record. Run 1, Event 4109, LumiSection 42 on stream 0 at 04-Sep-2023 10:45:13.072 CEST
Begin processing the 10th record. Run 1, Event 4110, LumiSection 42 on stream 0 at 04-Sep-2023 10:45:18.037 CEST
Begin processing the 11th record. Run 1, Event 4112, LumiSection 42 on stream 0 at 04-Sep-2023 10:45:23.611 CEST
Begin processing the 12th record. Run 1, Event 4111, LumiSection 42 on stream 0 at 04-Sep-2023 10:45:29.576 CEST
Begin processing the 13th record. Run 1, Event 4113, LumiSection 42 on stream 0 at 04-Sep-2023 10:45:34.543 CEST
Begin processing the 14th record. Run 1, Event 4114, LumiSection 42 on stream 0 at 04-Sep-2023 10:45:40.905 CEST
Begin processing the 15th record. Run 1, Event 4116, LumiSection 42 on stream 0 at 04-Sep-2023 10:45:45.968 CEST
Begin processing the 16th record. Run 1, Event 4115, LumiSection 42 on stream 0 at 04-Sep-2023 10:45:51.790 CEST
Begin processing the 17th record. Run 1, Event 4118, LumiSection 42 on stream 0 at 04-Sep-2023 10:45:56.942 CEST
Begin processing the 18th record. Run 1, Event 4117, LumiSection 42 on stream 0 at 04-Sep-2023 10:46:01.971 CEST
Begin processing the 19th record. Run 1, Event 4120, LumiSection 42 on stream 0 at 04-Sep-2023 10:46:06.970 CEST
Begin processing the 20th record. Run 1, Event 4119, LumiSection 42 on stream 0 at 04-Sep-2023 10:46:12.951 CEST
Begin processing the 21st record. Run 1, Event 4122, LumiSection 42 on stream 0 at 04-Sep-2023 10:46:17.937 CEST
Begin processing the 22nd record. Run 1, Event 4123, LumiSection 42 on stream 0 at 04-Sep-2023 10:46:23.614 CEST
Begin processing the 23rd record. Run 1, Event 4124, LumiSection 42 on stream 0 at 04-Sep-2023 10:46:28.440 CEST
Begin processing the 24th record. Run 1, Event 4121, LumiSection 42 on stream 0 at 04-Sep-2023 10:46:33.335 CEST
Begin processing the 25th record. Run 1, Event 4125, LumiSection 42 on stream 0 at 04-Sep-2023 10:46:39.215 CEST
Begin processing the 26th record. Run 1, Event 4126, LumiSection 42 on stream 0 at 04-Sep-2023 10:46:44.597 CEST
Begin processing the 27th record. Run 1, Event 4127, LumiSection 42 on stream 0 at 04-Sep-2023 10:46:50.107 CEST
Begin processing the 28th record. Run 1, Event 4128, LumiSection 42 on stream 0 at 04-Sep-2023 10:46:55.850 CEST
Begin processing the 29th record. Run 1, Event 4129, LumiSection 42 on stream 0 at 04-Sep-2023 10:47:00.418 CEST
Begin processing the 30th record. Run 1, Event 4130, LumiSection 42 on stream 0 at 04-Sep-2023 10:47:05.792 CEST
Begin processing the 31st record. Run 1, Event 4131, LumiSection 42 on stream 0 at 04-Sep-2023 10:47:11.826 CEST
Will break on number of tracklets in TP_L1L2F
cmsRun: /afs/cern.ch/user/d/dabraham/private/test/CMSSW_12_6_0_pre5/src/L1Trigger/TrackFindingTracklet/src/TrackletProcessor.cc:345: void trklet::TrackletProcessor::execute(unsigned int, double, double): Assertion `0' failed.


A fatal system signal has occurred: abort signal
The following is the call stack containing the origin of the signal.


Current Modules:

Module: L1FPGATrackProducer:l1tTTTracksFromTrackletEmulation (crashed)

A fatal system signal has occurred: abort signal
condor_exec.exe: line 7:   155 Aborted                 (core dumped) cmsRun L1TrackNtupleMaker_cfg_grid_NEWKF_trkJets.py inputFiles=root://cms-xrd-global.cern.ch///store/mc/CMSSW_12_6_0/RelValTTbar_14TeV/GEN-SIM-DIGI-RAW/PU_125X_mcRun4_realistic_v5_2026D88PU200RV183v2-v1/30000/4a62553b-ce6d-451a-865b-b50023fc4966.root outputFile=/eos/user/d/dabraham/L1NtupleTrackExamples/TTbarPU200D88_NEWKF_combined_noTrunc_assertsOn_4.root maxEvents=10000
