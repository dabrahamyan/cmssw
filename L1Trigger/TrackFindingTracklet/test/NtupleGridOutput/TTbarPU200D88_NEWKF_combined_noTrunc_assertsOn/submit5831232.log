000 (5831232.000.000) 09/04 10:34:38 Job submitted from host: <137.138.44.55:9618?addrs=137.138.44.55-9618+[2001-1458-d00-1a--48]-9618&alias=bigbird17.cern.ch&noUDP&sock=schedd_7840_2a97>
...
022 (5831232.000.000) 09/04 10:36:01 Job disconnected, attempting to reconnect
    Socket between submit and execute hosts closed unexpectedly
    Trying to reconnect to slot1_24@b7s11p5714.cern.ch <188.184.162.216:9618?addrs=188.184.162.216-9618+[2001-1458-301-4a--100-d2]-9618&alias=b7s11p5714.cern.ch&noUDP&sock=startd_3002_b366>
...
024 (5831232.000.000) 09/04 10:36:01 Job reconnection failed
    Job not found at execution machine
    Can not reconnect to slot1_24@b7s11p5714.cern.ch, rescheduling job
...
022 (5831232.000.000) 09/04 10:42:00 Job disconnected, attempting to reconnect
    Socket between submit and execute hosts closed unexpectedly
    Trying to reconnect to slot1_7@b7s02p9794.cern.ch <128.142.57.36:9618?addrs=128.142.57.36-9618+[2001-1458-301-7--100-23]-9618&alias=b7s02p9794.cern.ch&noUDP&sock=startd_4809_2708>
...
024 (5831232.000.000) 09/04 10:42:00 Job reconnection failed
    Job not found at execution machine
    Can not reconnect to slot1_7@b7s02p9794.cern.ch, rescheduling job
...
040 (5831232.000.000) 09/04 10:43:33 Started transferring input files
	Transferring to host: <188.184.163.63:9618?addrs=188.184.163.63-9618+[2001-1458-301-4d--100-39]-9618&alias=b7s12p8378.cern.ch&noUDP&sock=slot1_5_4366_fdeb_51595>
...
040 (5831232.000.000) 09/04 10:43:33 Finished transferring input files
...
001 (5831232.000.000) 09/04 10:43:34 Job executing on host: <188.184.163.63:9618?addrs=188.184.163.63-9618+[2001-1458-301-4d--100-39]-9618&alias=b7s12p8378.cern.ch&noUDP&sock=startd_3135_ed7e>
...
006 (5831232.000.000) 09/04 10:43:43 Image size of job updated: 185212
	181  -  MemoryUsage of job (MB)
	185212  -  ResidentSetSize of job (KB)
...
006 (5831232.000.000) 09/04 10:47:25 Image size of job updated: 7335648
	181  -  MemoryUsage of job (MB)
	185212  -  ResidentSetSize of job (KB)
...
040 (5831232.000.000) 09/04 10:47:25 Started transferring output files
...
040 (5831232.000.000) 09/04 10:47:25 Finished transferring output files
...
005 (5831232.000.000) 09/04 10:47:25 Job terminated.
	(1) Normal termination (return value 134)
		Usr 0 00:03:03, Sys 0 00:00:11  -  Run Remote Usage
		Usr 0 00:00:00, Sys 0 00:00:00  -  Run Local Usage
		Usr 0 00:03:03, Sys 0 00:00:11  -  Total Remote Usage
		Usr 0 00:00:00, Sys 0 00:00:00  -  Total Local Usage
	23772  -  Run Bytes Sent By Job
	696  -  Run Bytes Received By Job
	23772  -  Total Bytes Sent By Job
	696  -  Total Bytes Received By Job
	Partitionable Resources :    Usage  Request Allocated 
	   Cpus                 :                 1         1 
	   Disk (KB)            :       39        1   2928281 
	   Memory (MB)          :      181     2000      2000 

	Job terminated of its own accord at 2023-09-04T08:47:25Z.
...
