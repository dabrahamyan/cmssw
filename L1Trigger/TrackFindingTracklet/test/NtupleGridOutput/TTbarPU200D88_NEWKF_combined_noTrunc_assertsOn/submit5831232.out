using geometry D88 (tilted)
Mon Sep  4 10:47:14 CEST 2023
Thread 7 (Thread 0x2ab7bbba1700 (LWP 184) "cmsRun"):
#0  0x00002ab788639b3b in do_futex_wait.constprop () from /lib64/libpthread.so.0
#1  0x00002ab788639bcf in __new_sem_wait_slow.constprop.0 () from /lib64/libpthread.so.0
#2  0x00002ab788639c6b in sem_wait@@GLIBC_2.2.5 () from /lib64/libpthread.so.0
#3  0x00002ab78f11f576 in XrdCl::JobManager::RunJobs() () from /cvmfs/cms.cern.ch/slc7_amd64_gcc10/cms/cmssw/CMSSW_12_6_0_pre5/external/slc7_amd64_gcc10/lib/libXrdCl.so.3
#4  0x00002ab78f11f639 in RunRunnerThread () from /cvmfs/cms.cern.ch/slc7_amd64_gcc10/cms/cmssw/CMSSW_12_6_0_pre5/external/slc7_amd64_gcc10/lib/libXrdCl.so.3
#5  0x00002ab788633ea5 in start_thread () from /lib64/libpthread.so.0
#6  0x00002ab788946b0d in clone () from /lib64/libc.so.6
Thread 6 (Thread 0x2ab7bb9a0700 (LWP 183) "cmsRun"):
#0  0x00002ab788639b3b in do_futex_wait.constprop () from /lib64/libpthread.so.0
#1  0x00002ab788639bcf in __new_sem_wait_slow.constprop.0 () from /lib64/libpthread.so.0
#2  0x00002ab788639c6b in sem_wait@@GLIBC_2.2.5 () from /lib64/libpthread.so.0
#3  0x00002ab78f11f576 in XrdCl::JobManager::RunJobs() () from /cvmfs/cms.cern.ch/slc7_amd64_gcc10/cms/cmssw/CMSSW_12_6_0_pre5/external/slc7_amd64_gcc10/lib/libXrdCl.so.3
#4  0x00002ab78f11f639 in RunRunnerThread () from /cvmfs/cms.cern.ch/slc7_amd64_gcc10/cms/cmssw/CMSSW_12_6_0_pre5/external/slc7_amd64_gcc10/lib/libXrdCl.so.3
#5  0x00002ab788633ea5 in start_thread () from /lib64/libpthread.so.0
#6  0x00002ab788946b0d in clone () from /lib64/libc.so.6
Thread 5 (Thread 0x2ab7bb79f700 (LWP 182) "cmsRun"):
#0  0x00002ab788639b3b in do_futex_wait.constprop () from /lib64/libpthread.so.0
#1  0x00002ab788639bcf in __new_sem_wait_slow.constprop.0 () from /lib64/libpthread.so.0
#2  0x00002ab788639c6b in sem_wait@@GLIBC_2.2.5 () from /lib64/libpthread.so.0
#3  0x00002ab78f11f576 in XrdCl::JobManager::RunJobs() () from /cvmfs/cms.cern.ch/slc7_amd64_gcc10/cms/cmssw/CMSSW_12_6_0_pre5/external/slc7_amd64_gcc10/lib/libXrdCl.so.3
#4  0x00002ab78f11f639 in RunRunnerThread () from /cvmfs/cms.cern.ch/slc7_amd64_gcc10/cms/cmssw/CMSSW_12_6_0_pre5/external/slc7_amd64_gcc10/lib/libXrdCl.so.3
#5  0x00002ab788633ea5 in start_thread () from /lib64/libpthread.so.0
#6  0x00002ab788946b0d in clone () from /lib64/libc.so.6
Thread 4 (Thread 0x2ab7bb59e700 (LWP 181) "cmsRun"):
#0  0x00002ab78863ae9d in nanosleep () from /lib64/libpthread.so.0
#1  0x00002ab78ee5c10f in XrdSysTimer::Wait(int) () from /cvmfs/cms.cern.ch/slc7_amd64_gcc10/cms/cmssw/CMSSW_12_6_0_pre5/external/slc7_amd64_gcc10/lib/libXrdUtils.so.3
#2  0x00002ab78f08d2d3 in XrdCl::TaskManager::RunTasks() () from /cvmfs/cms.cern.ch/slc7_amd64_gcc10/cms/cmssw/CMSSW_12_6_0_pre5/external/slc7_amd64_gcc10/lib/libXrdCl.so.3
#3  0x00002ab78f08d429 in RunRunnerThread () from /cvmfs/cms.cern.ch/slc7_amd64_gcc10/cms/cmssw/CMSSW_12_6_0_pre5/external/slc7_amd64_gcc10/lib/libXrdCl.so.3
#4  0x00002ab788633ea5 in start_thread () from /lib64/libpthread.so.0
#5  0x00002ab788946b0d in clone () from /lib64/libc.so.6
Thread 3 (Thread 0x2ab7bb39d700 (LWP 180) "cmsRun"):
#0  0x00002ab7889470e3 in epoll_wait () from /lib64/libc.so.6
#1  0x00002ab78ee565c2 in XrdSys::IOEvents::PollE::Begin(XrdSysSemaphore*, int&, char const**) () from /cvmfs/cms.cern.ch/slc7_amd64_gcc10/cms/cmssw/CMSSW_12_6_0_pre5/external/slc7_amd64_gcc10/lib/libXrdUtils.so.3
#2  0x00002ab78ee52d4d in XrdSys::IOEvents::BootStrap::Start(void*) () from /cvmfs/cms.cern.ch/slc7_amd64_gcc10/cms/cmssw/CMSSW_12_6_0_pre5/external/slc7_amd64_gcc10/lib/libXrdUtils.so.3
#3  0x00002ab78ee5b818 in XrdSysThread_Xeq () from /cvmfs/cms.cern.ch/slc7_amd64_gcc10/cms/cmssw/CMSSW_12_6_0_pre5/external/slc7_amd64_gcc10/lib/libXrdUtils.so.3
#4  0x00002ab788633ea5 in start_thread () from /lib64/libpthread.so.0
#5  0x00002ab788946b0d in clone () from /lib64/libc.so.6
Thread 2 (Thread 0x2ab7abb26700 (LWP 173) "cmsRun"):
#0  0x00002ab78863b1d9 in waitpid () from /lib64/libpthread.so.0
#1  0x00002ab78f79cad7 in edm::service::cmssw_stacktrace_fork() () from /cvmfs/cms.cern.ch/slc7_amd64_gcc10/cms/cmssw/CMSSW_12_6_0_pre5/lib/slc7_amd64_gcc10/pluginFWCoreServicesPlugins.so
#2  0x00002ab78f79d6ca in edm::service::InitRootHandlers::stacktraceHelperThread() () from /cvmfs/cms.cern.ch/slc7_amd64_gcc10/cms/cmssw/CMSSW_12_6_0_pre5/lib/slc7_amd64_gcc10/pluginFWCoreServicesPlugins.so
#3  0x00002ab78820ff90 in std::execute_native_thread_routine (__p=0x2ab797750060) at ../../../../../libstdc++-v3/src/c++11/thread.cc:80
#4  0x00002ab788633ea5 in start_thread () from /lib64/libpthread.so.0
#5  0x00002ab788946b0d in clone () from /lib64/libc.so.6
Thread 1 (Thread 0x2ab78a9c9e40 (LWP 155) "cmsRun"):
#0  0x00002ab78893bddd in poll () from /lib64/libc.so.6
#1  0x00002ab78f79cdcf in full_read.constprop () from /cvmfs/cms.cern.ch/slc7_amd64_gcc10/cms/cmssw/CMSSW_12_6_0_pre5/lib/slc7_amd64_gcc10/pluginFWCoreServicesPlugins.so
#2  0x00002ab78f79d79c in edm::service::InitRootHandlers::stacktraceFromThread() () from /cvmfs/cms.cern.ch/slc7_amd64_gcc10/cms/cmssw/CMSSW_12_6_0_pre5/lib/slc7_amd64_gcc10/pluginFWCoreServicesPlugins.so
#3  0x00002ab78f7a024b in sig_dostack_then_abort () from /cvmfs/cms.cern.ch/slc7_amd64_gcc10/cms/cmssw/CMSSW_12_6_0_pre5/lib/slc7_amd64_gcc10/pluginFWCoreServicesPlugins.so
#4  <signal handler called>
#5  0x00002ab78887e387 in raise () from /lib64/libc.so.6
#6  0x00002ab78887fa78 in abort () from /lib64/libc.so.6
#7  0x00002ab7888771a6 in __assert_fail_base () from /lib64/libc.so.6
#8  0x00002ab788877252 in __assert_fail () from /lib64/libc.so.6
#9  0x00002ab7b8fb4331 in trklet::TrackletProcessor::execute(unsigned int, double, double) () from /afs/cern.ch/user/d/dabraham/private/test/CMSSW_12_6_0_pre5/lib/slc7_amd64_gcc10/libL1TriggerTrackFindingTracklet.so
#10 0x00002ab7b8f30b59 in trklet::Sector::executeTP() () from /afs/cern.ch/user/d/dabraham/private/test/CMSSW_12_6_0_pre5/lib/slc7_amd64_gcc10/libL1TriggerTrackFindingTracklet.so
#11 0x00002ab7b8fa1263 in trklet::TrackletEventProcessor::event(trklet::SLHCEvent&, std::vector<std::vector<std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> >, std::allocator<std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > > >, std::allocator<std::vector<std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> >, std::allocator<std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > > > > >&, std::vector<std::vector<trklet::StubStreamData, std::allocator<trklet::StubStreamData> >, std::allocator<std::vector<trklet::StubStreamData, std::allocator<trklet::StubStreamData> > > >&) () from /afs/cern.ch/user/d/dabraham/private/test/CMSSW_12_6_0_pre5/lib/slc7_amd64_gcc10/libL1TriggerTrackFindingTracklet.so
#12 0x00002ab7b8ddba74 in L1FPGATrackProducer::produce(edm::Event&, edm::EventSetup const&) () from /afs/cern.ch/user/d/dabraham/private/test/CMSSW_12_6_0_pre5/lib/slc7_amd64_gcc10/pluginTrackFindingTrackletPlugins.so
#13 0x00002ab7862569ae in edm::one::EDProducerBase::doEvent(edm::EventTransitionInfo const&, edm::ActivityRegistry*, edm::ModuleCallingContext const*) () from /cvmfs/cms.cern.ch/slc7_amd64_gcc10/cms/cmssw/CMSSW_12_6_0_pre5/lib/slc7_amd64_gcc10/libFWCoreFramework.so
#14 0x00002ab78624023f in edm::WorkerT<edm::one::EDProducerBase>::implDo(edm::EventTransitionInfo const&, edm::ModuleCallingContext const*) () from /cvmfs/cms.cern.ch/slc7_amd64_gcc10/cms/cmssw/CMSSW_12_6_0_pre5/lib/slc7_amd64_gcc10/libFWCoreFramework.so
#15 0x00002ab786190b25 in decltype ({parm#1}()) edm::convertException::wrap<edm::Worker::runModule<edm::OccurrenceTraits<edm::EventPrincipal, (edm::BranchActionType)1> >(edm::OccurrenceTraits<edm::EventPrincipal, (edm::BranchActionType)1>::TransitionInfoType const&, edm::StreamID, edm::ParentContext const&, edm::OccurrenceTraits<edm::EventPrincipal, (edm::BranchActionType)1>::Context const*)::{lambda()#1}>(edm::Worker::runModule<edm::OccurrenceTraits<edm::EventPrincipal, (edm::BranchActionType)1> >(edm::OccurrenceTraits<edm::EventPrincipal, (edm::BranchActionType)1>::TransitionInfoType const&, edm::StreamID, edm::ParentContext const&, edm::OccurrenceTraits<edm::EventPrincipal, (edm::BranchActionType)1>::Context const*)::{lambda()#1}) () from /cvmfs/cms.cern.ch/slc7_amd64_gcc10/cms/cmssw/CMSSW_12_6_0_pre5/lib/slc7_amd64_gcc10/libFWCoreFramework.so
#16 0x00002ab786190e13 in std::__exception_ptr::exception_ptr edm::Worker::runModuleAfterAsyncPrefetch<edm::OccurrenceTraits<edm::EventPrincipal, (edm::BranchActionType)1> >(std::__exception_ptr::exception_ptr, edm::OccurrenceTraits<edm::EventPrincipal, (edm::BranchActionType)1>::TransitionInfoType const&, edm::StreamID, edm::ParentContext const&, edm::OccurrenceTraits<edm::EventPrincipal, (edm::BranchActionType)1>::Context const*) () from /cvmfs/cms.cern.ch/slc7_amd64_gcc10/cms/cmssw/CMSSW_12_6_0_pre5/lib/slc7_amd64_gcc10/libFWCoreFramework.so
#17 0x00002ab786193b5a in void edm::SerialTaskQueueChain::actionToRun<edm::Worker::RunModuleTask<edm::OccurrenceTraits<edm::EventPrincipal, (edm::BranchActionType)1> >::execute()::{lambda()#1}&>(edm::Worker::RunModuleTask<edm::OccurrenceTraits<edm::EventPrincipal, (edm::BranchActionType)1> >::execute()::{lambda()#1}&) () from /cvmfs/cms.cern.ch/slc7_amd64_gcc10/cms/cmssw/CMSSW_12_6_0_pre5/lib/slc7_amd64_gcc10/libFWCoreFramework.so
#18 0x00002ab786193dd1 in edm::SerialTaskQueue::QueuedTask<edm::SerialTaskQueueChain::push<edm::Worker::RunModuleTask<edm::OccurrenceTraits<edm::EventPrincipal, (edm::BranchActionType)1> >::execute()::{lambda()#1}&>(tbb::detail::d1::task_group&, edm::Worker::RunModuleTask<edm::OccurrenceTraits<edm::EventPrincipal, (edm::BranchActionType)1> >::execute()::{lambda()#1}&)::{lambda()#1}>::execute() () from /cvmfs/cms.cern.ch/slc7_amd64_gcc10/cms/cmssw/CMSSW_12_6_0_pre5/lib/slc7_amd64_gcc10/libFWCoreFramework.so
#19 0x00002ab785f2b175 in tbb::detail::d1::function_task<edm::SerialTaskQueue::spawn(edm::SerialTaskQueue::TaskBase&)::{lambda()#1}>::execute(tbb::detail::d1::execution_data&) () from /cvmfs/cms.cern.ch/slc7_amd64_gcc10/cms/cmssw/CMSSW_12_6_0_pre5/lib/slc7_amd64_gcc10/libFWCoreConcurrency.so
#20 0x00002ab787a26bec in tbb::detail::r1::task_dispatcher::local_wait_for_all<false, tbb::detail::r1::external_waiter> (waiter=..., t=0x2ab78c0d6b00, this=0x2ab78c0cbe00) at /data/cmsbld/jenkins/workspace/auto-builds/CMSSW_12_6_0_pre3-slc7_amd64_gcc10/build/CMSSW_12_6_0_pre3-build/BUILD/slc7_amd64_gcc10/external/tbb/v2021.5.0-36aff7df349e0716374b1668ccd18e17/tbb-v2021.5.0/src/tbb/task_dispatcher.h:322
#21 tbb::detail::r1::task_dispatcher::local_wait_for_all<tbb::detail::r1::external_waiter> (waiter=..., t=<optimized out>, this=0x2ab78c0cbe00) at /data/cmsbld/jenkins/workspace/auto-builds/CMSSW_12_6_0_pre3-slc7_amd64_gcc10/build/CMSSW_12_6_0_pre3-build/BUILD/slc7_amd64_gcc10/external/tbb/v2021.5.0-36aff7df349e0716374b1668ccd18e17/tbb-v2021.5.0/src/tbb/task_dispatcher.h:463
#22 tbb::detail::r1::task_dispatcher::execute_and_wait (t=<optimized out>, wait_ctx=..., w_ctx=...) at /data/cmsbld/jenkins/workspace/auto-builds/CMSSW_12_6_0_pre3-slc7_amd64_gcc10/build/CMSSW_12_6_0_pre3-build/BUILD/slc7_amd64_gcc10/external/tbb/v2021.5.0-36aff7df349e0716374b1668ccd18e17/tbb-v2021.5.0/src/tbb/task_dispatcher.cpp:168
#23 0x00002ab78610b22d in edm::FinalWaitingTask::wait() () from /cvmfs/cms.cern.ch/slc7_amd64_gcc10/cms/cmssw/CMSSW_12_6_0_pre5/lib/slc7_amd64_gcc10/libFWCoreFramework.so
#24 0x00002ab7860f3310 in edm::EventProcessor::processRuns() () from /cvmfs/cms.cern.ch/slc7_amd64_gcc10/cms/cmssw/CMSSW_12_6_0_pre5/lib/slc7_amd64_gcc10/libFWCoreFramework.so
#25 0x00002ab7860ffda1 in edm::EventProcessor::runToCompletion() () from /cvmfs/cms.cern.ch/slc7_amd64_gcc10/cms/cmssw/CMSSW_12_6_0_pre5/lib/slc7_amd64_gcc10/libFWCoreFramework.so
#26 0x000000000040a266 in tbb::detail::d1::task_arena_function<main::{lambda()#1}::operator()() const::{lambda()#1}, void>::operator()() const ()
#27 0x00002ab787a150eb in tbb::detail::r1::task_arena_impl::execute (ta=..., d=...) at /data/cmsbld/jenkins/workspace/auto-builds/CMSSW_12_6_0_pre3-slc7_amd64_gcc10/build/CMSSW_12_6_0_pre3-build/BUILD/slc7_amd64_gcc10/external/tbb/v2021.5.0-36aff7df349e0716374b1668ccd18e17/tbb-v2021.5.0/src/tbb/arena.cpp:698
#28 0x000000000040b094 in main::{lambda()#1}::operator()() const ()
#29 0x000000000040971c in main ()
